utils/qemu/Makefile

```diff
@@ -378,6 +377,7 @@ CONFIGURE_ARGS +=			\
 #
 CONFIGURE_ARGS +=			\
 	--audio-drv-list=''		\
+	--disable-af-xdp                \
```

Comment:
This is not said in the commit.

Reply:
By defaul most of the fatures if disable in openwrt, so i decided to to enable feature we didn't use previosly.

Reply:
Okay so please mention it in your commit message like "disabled new options introduced in this version"

Reply:
@1715173329 done.